scripts/init.sh

```diff
@@ -11,11 +11,18 @@ pushd  polkadot-sdk
 git checkout snowbridge
 popd
 popd
-
 ln -sf ../polkadot-sdk polkadot-sdk
 
-echo "Setting up submodules"
-git submodule update --init --recursive || true
+echo "Checkout lodestar Snowfork fork"
+pushd ..
+  if [ ! -d "lodestar" ]; then
+    git clone https://github.com/Snowfork/lodestar
```

Comment:
We are using Lodestar 1.16.0 in prod. Can we not use this version using npx? I think the condition we had to build Lodestar ourselves was for 1.12.0.

Reply:
> Can we not use this version using npx?
Error as follows 
```
test git:(ron/skip-mandatory-commitment) ✗ npx lodestar dev \
            --genesisValidators 8 \
            --genesisTime $timestamp \
            --startValidators "0..7" \
            --enr.ip6 "127.0.0.1" \
            --eth1.providerUrls "http://127.0.0.1:8545" \
            --execution.urls "http://127.0.0.1:8551" \
            --dataDir "$ethereum_data_dir" \
            --reset \
            --terminal-total-difficulty-override 0 \
            --genesisEth1Hash $genesisHash \
            --params.ALTAIR_FORK_EPOCH 0 \
            --params.BELLATRIX_FORK_EPOCH 0 \
            --params.CAPELLA_FORK_EPOCH 0 \
            --params.DENEB_FORK_EPOCH 0 \
            --eth1=true \
            --rest.namespace="*" \
            --jwt-secret $config_dir/jwtsecret \
            --chain.archiveStateEpochFrequency 1
Eph 0/1 10.305[]                 info: Lodestar network=dev, version=v1.18.1/ron/skip-mandatory-commitment/40f9592, commit=40f95923a816184dfb39c8e1e30a89b4f8d1c0f1
Eph 0/1 10.310[]                 info: Connected to LevelDB database path=/tmp/snowbridge/ethereum/chain-db
Eph 0/1 10.414[]                 info: Initializing beacon from a valid checkpoint state slot=0, epoch=0, stateRoot=0x36da8093ea5e145f6d10fe0bc2f7af2ec492635bd191bd6c6aeecc8fb483eb25, isWithinWeakSubjectivityPeriod=true
 ✖ Error: Error loading trusted setup /Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/@lodestar+beacon-node@1.18.1/node_modules/@lodestar/beacon-node/trusted_setup.json: no trusted setup found: /Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/@lodestar+beacon-node@1.18.1/node_modules/@lodestar/beacon-node/trusted_setup.txt
    at Object.loadTrustedSetup (/Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/c-kzg@2.1.2/node_modules/c-kzg/lib/kzg.js:41:11)
    at loadEthereumTrustedSetup (file:///Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/@lodestar+beacon-node@1.18.1/node_modules/@lodestar/beacon-node/src/util/kzg.ts:78:12)
    at Function.init (file:///Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/@lodestar+beacon-node@1.18.1/node_modules/@lodestar/beacon-node/src/node/nodejs.ts:163:7)
    at beaconHandler (file:///Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/@chainsafe+lodestar@1.18.1/node_modules/@chainsafe/lodestar/src/cmds/beacon/handler.ts:76:18)
    at Object.devHandler [as handler] (file:///Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/@chainsafe+lodestar@1.18.1/node_modules/@chainsafe/lodestar/src/cmds/dev/handler.ts:73:3)
```

Reply:
> We are using Lodestar 1.16.0 in prod.

Update the forked https://github.com/Snowfork/lodestar snowbridge branch to v1.16.0

Reply:
That error looks like it might be a setup error? https://github.com/ChainSafe/lodestar/blob/8b6ecc44c1892115d0e7396b2a5f0ae8fc0d9e6f/packages/beacon-node/src/util/kzg.ts#L62 What happens if you create that trusted file? I think it worthwhile to see if we can do away with maintaining our own lodestar fork.

Reply:
After manually copy the trusted file to destination that error resolved, but it raise another error as follows:

```
Eph 0/0 9.774[]                 info: Lodestar network=dev, version=v1.18.1/ron/remove-lodestar-from-submodule/586fd58, commit=586fd58e2e136b3736c39dff56831ca77d0f6614
Eph 0/0 9.779[]                 info: Connected to LevelDB database path=/tmp/snowbridge/ethereum/chain-db
Eph 0/0 9.883[]                 info: Initializing beacon from a valid checkpoint state slot=0, epoch=0, stateRoot=0xca779bfd9e71b909705795374e7817ab1e886ec73db678447a3ccbaf4be1c6dd, isWithinWeakSubjectivityPeriod=true
Eph 0/0 10.012[eth1]             info: Eth1 provider urls=http://127.0.0.1:8545
Eph 0/0 10.029[execution]        info: Execution client urls=http://127.0.0.1:8551
Eph 0/0 10.058[network]          info: running libp2p instance in worker thread
Eph 0/0 10.633[network]          info: libp2p worker started peer=16Uiu2HAkuZt6LG3Szo6BiSnTFpRpV94kbvgg2YNKDLK77Cx4LS8P
Eph 0/0 11.029[network]         error: PeerDiscovery: discv5 has no boot enr
Eph 0/0 11.033[network]          info: PeerId 16Uiu2HAkuZt6LG3Szo6BiSnTFpRpV94kbvgg2YNKDLK77Cx4LS8P, Multiaddrs /ip4/0.0.0.0/tcp/9000
Eph 0/0 11.028[network]          info: discv5 worker started peerId=16Uiu2HAkuZt6LG3Szo6BiSnTFpRpV94kbvgg2YNKDLK77Cx4LS8P, initialENR=enr:-Iy4QAAoH0LNbXOvn_GvxByRTMAPGNDTk1Te726_-_s85XhdU3Bgb9efKpm75gDNkc3crItw8eC20lH3dRDPX3dxrfcBgmlkgnY0g2lwNoR_AAABiXNlY3AyNTZrMaECAhpXp4UhP_XdWEUAlkEBfkdUf7LLwQrAVU8VlZ158XiDdGNwgiMog3VkcIIjKA, bindAddr4=/ip4/0.0.0.0/udp/9000
Eph 0/0 11.118[]                error: uncaughtException: Cannot read properties of undefined (reading 'content')
TypeError: Cannot read properties of undefined (reading 'content')
    at swaggerInitializer (/Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/@fastify+swagger-ui@1.10.2/node_modules/@fastify/swagger-ui/lib/swagger-initializer.js:6:44)
    at fastifySwagger (/Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/@fastify+swagger-ui@1.10.2/node_modules/@fastify/swagger-ui/lib/routes.js:144:37)
    at Plugin.exec (/Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/avvio@8.3.0/node_modules/avvio/lib/plugin.js:122:28)
    at Boot._loadPlugin (/Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/avvio@8.3.0/node_modules/avvio/boot.js:423:10)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) - Cannot read properties of undefined (reading 'content')
TypeError: Cannot read properties of undefined (reading 'content')
    at swaggerInitializer (/Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/@fastify+swagger-ui@1.10.2/node_modules/@fastify/swagger-ui/lib/swagger-initializer.js:6:44)
    at fastifySwagger (/Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/@fastify+swagger-ui@1.10.2/node_modules/@fastify/swagger-ui/lib/routes.js:144:37)
    at Plugin.exec (/Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/avvio@8.3.0/node_modules/avvio/lib/plugin.js:122:28)
    at Boot._loadPlugin (/Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/avvio@8.3.0/node_modules/avvio/boot.js:423:10)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
 ✖ FastifyError: Plugin did not start in time: 'fastifySwagger'. You may have forgotten to call 'done' function or to resolve a Promise
    at Timeout._onTimeout (/Users/yangrong/Projects/snowbridge/web/node_modules/.pnpm/avvio@8.3.0/node_modules/avvio/lib/plugin.js:112:31)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7)
```

As mentioned in https://chainsafe.github.io/lodestar/getting-started/installation, only 2 ways officially support either from docker or build from source with yarn&lerna.  It seems `npx` is not officially supported any more.

Reply:
Btw, agree we can just use source code from the official repo without fork, https://github.com/Snowfork/snowbridge/pull/1212/commits/772a68d6ede05556e8650e28f83e0f37ddcd8ffa accordingly.

Reply:
Cool! :D